packages/web/components/cards/stake-info-card.tsx

```diff
@@ -54,13 +54,17 @@ export const StakeInfoCard: FunctionComponent<{
 
   const handleInputChange = useCallback(
     (e: React.ChangeEvent<HTMLInputElement>) => {
+      let value = e.target.value;
+      if (value.startsWith(".")) {
+        value = "0" + value;
+      }
```

Comment:
I'm really not OK with us not using an AmountConfig to manage the user input.

Doing otherwise is error prone.

Reply:
Once the previews work I'll check out the amount UX